jwst/datamodels/steppars.py

```diff
@@ -25,10 +25,9 @@ class StepParsModel(DataModel):
     A data model for `Step` parameters.
     """
     schema_url = "steppars.schema"
-    supported_formats = ['yaml', 'json', 'asdf']
 
     def __init__(self, init=None, **kwargs):
-        super().__init__(init=init, **kwargs)
+        super(StepParsModel, self).__init__(init=init, **kwargs)
```

Comment:
This is unnecessary since python 3.  The original implementation is fine.

Reply:
ya, i was just trying this to see jp-1041 was affected by this. However, I have since figured out what is the basis of that issue.